app/views/facilities/_product_list.html.haml

```diff
@@ -10,17 +10,15 @@
     %ul
       - indent_for_icon = products.any? { |p| p.respond_to? :reservations }
       - products.sort.each do |product|
-        %li{ class: product.class.model_name.to_s.downcase }
-          - if local_assigns[:f]
-            = f.fields_for :order_details do |builder|
-              - if session_user.can_override_restrictions?(product) || product.can_be_used_by?(acting_user)
-                = builder.text_field :quantity, value: 0, class: "product_quantity", index: nil
-                = builder.hidden_field :product_id, value: product.id, index: nil
-          = public_calendar_link(product, indent: indent_for_icon)
-          = link_to product.name + (product.is_hidden? ? ' (hidden)' : ''), facility_product_path(current_facility || product.facility, product)
-          - if acting_user.present? && !product.can_be_used_by?(acting_user)
-            %i.fa.fa-lock
-            = " (#{product.class.human_attribute_name(:requires_approval_show)})"
-
-          - if product.offline?
-            = tooltip_icon "fa fa-exclamation-triangle icon-large", t("instruments.offline.note")
+        - if quicklinks?
+          %li{ class: product.class.model_name.to_s.downcase + '_title' }
+          = render :partial => "facilities/quicklinks", locals: {product: product, indent_for_icon: indent_for_icon}
+        - else
```

Comment:
I think some of the original code got lost here. The `else` part of this should be the original code, correct? We've definitely lost all of the icons, and the order form input is now to the right of the name where previously it was to the left.